sass/px-dropdown.scss

```diff
@@ -29,21 +25,21 @@ $inuit-enable-btn--disabled: true;
 $inuit-enable-btn--bare: true;
 $inuit-enable-btn--bare--primary: true;
 @import "px-buttons-design/_objects.buttons.scss";
+@import "px-shadows-design/_objects.shadows.scss";
 
 // Component
 :host {
   display: flex;
   position: relative;
-  text-align: left;
-  -webkit-tap-highlight-color: rgba(0,0,0,0);
+  // -webkit-tap-highlight-color: rgba(0,0,0,0);// <-- Why is this defined twice
```

Comment:
I know it's an accessibility issue, but Chrome is also the only browser that uses `outline` - all of the others have even less affordance (I think IE might have a very subtle dotted line). The designers don't like the default `outline` behavior in Chrome, I think I've noted in some of the other repos where I've set `outline: none` that we need a system-wide cross-browser affordance for highlighting current focus (maybe focus-ring is that answer).